bbserver/bb_server.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask
+import edn_format as edn
```

Comment:
`edn` is a data format, ala `json`, `xml`, ...
`edn` is the clojure datastructure